test/views/EditSetRecordEditDiscardSpec.scala

```diff
@@ -57,35 +55,33 @@ class EditSetRecordEditDiscardSpec extends BaseSpec {
         )
       )
       document must haveBackLink("/edit-set/1/record/EditRecordOciTest/edit", "edit-set.record.save.back")
-
     }
 
-    "render the header" in {
-
-      val document = generateDocument()
+    "render the header" in new TestCase {
+      val confirmationEditSetRecordEditHtml = editSetRecordEditDiscardInstance(
+        user = user,
+        editSetName = "COAL 80 Sample",
+        title = "EditRecordTitleTest",
+        heading = "EditRecordHeadingTest",
+        oci = "EditRecordOciTest",
+        message = "Any changes have been discarded. Showing last saved version.",
+        recordType = Some(PhysicalRecord)
+      )
+      val document = asDocument(confirmationEditSetRecordEditHtml)
 
       document must haveHeaderTitle("header.title")
       document must haveVisibleLogoutLink
       document must haveLogoutLinkLabel("header.logout")
       document must haveLogoutLink
 
     }
+
   }
 
-  private def generateDocument(): Document = {
+  class TestCase() {
```

Comment:
I don't mind this pattern, but it isn't consistent with most of the other specs which use class level members. 

Should we adopt this approach, consistently? 

Let's remember that - even without this fixture - each scenario gets a new instance of the class, and thus members.

Reply:
I think that's fine to have them top level, I'll make that change.

Reply:
Confirmed.